doc/src/base/collections.md

```diff
@@ -235,6 +236,56 @@ Partially implemented by:
   * [`Iterators.Pairs`](@ref)
 
 ## Set-Like Collections
+A [`Set`](@ref) is a collection of elements, just like an array or dictionary, with no duplicated elements.
```

Comment:
It is a bit different from arrays or dictionaries though, since `Set`s have no keys, or?